test/features/data_qld_theme.feature

```diff
@@ -1,11 +1,11 @@
-@data-qld-theme
-Feature: Data QLD Theme
+@publications-qld-theme
+Feature: Publications QLD Theme
 
        Scenario: Lato font is implemented on homepage
               When I go to homepage
               Then I should see an element with xpath "//link[contains(@href,'https://fonts.googleapis.com/css?family=Lato')]"
 
-       Scenario: Organisation is in fact spelled Organisation (as opposed to Organization) 
+       Scenario: Organisation is in fact spelled Organisation (as opposed to Organization)
```

Comment:
can we get a test to prove that dataset is renamed  Publication i.e  Publication sets vs Data etc
per current theme.
https://www.publications.qld.gov.au/dataset

Reply:
The test fails, actually; there is no title above the search bar in the new theme.